CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Increment the:
 
 ## [1.0.1] 2021-10-21
 
+* [EXPORTER] Prometheus Exporter ([#1031](https://github.com/open-telemetry/opentelemetry-cpp/pull/1031))
```

Comment:
This should be put into the above section for "Unreleased".

Reply:
done